OpenRA.Mods.Common/Widgets/Logic/ConnectionLogic.cs

```diff
@@ -142,4 +142,4 @@ public ConnectionFailedLogic(Widget widget, OrderManager orderManager, Action on
 			}
 		}
 	}
-}
\ No newline at end of file
+}
```

Comment:
There are various automatic processes that like to fight over the end-of-file whitespace style, so our policy is for PRs to leave them as they found them, unless the pr is specifically dealing with that on a project-wide basis :)